src/Bio/VCF/Internal/Types.hs

```diff
@@ -58,15 +60,15 @@ instance ToJSON Header
 
 
 data Variation = Variation
-              { chrom  :: ByteString
-              , pos    :: Int
-              , idx    :: [ByteString]
-              , ref    :: ByteString
-              , alt    :: [ByteString]
-              , qual   :: Maybe Float
-              , filt   :: [ByteString]
-              , info   :: [ByteString]
-              , format :: Maybe [ByteString]
+              { chrom  :: !ByteString
```

Comment:
What makes these `ByteString` different from each other? Maybe you can also add some more descriptive types for them.